py/dml/ctree.py

```diff
@@ -196,7 +196,6 @@ def expr(self, site):
         expr.incref()
         return expr
 
-# HACK this exists to make a hack in traits.Trait.scope possible
 class InvalidExpressionSymbol(symtab.Symbol):
     """A symbol that corresponds to an invalid expression"""
     def __init__(self, name, mkError, site):
```

Comment:
`make_error`

Reply:
also, PEP8 asks you to say `site=site`, without spaces, in the kw args of the `super` call.